.madrun.js

```diff
@@ -23,7 +23,7 @@ const names = [
     '{client,server,common}/**/*.spec.js',
 ];
 
-const {putout} = predefined;
+const { putout } = predefined;
```

Comment:
Could you please run `npm run fix:lint` :)

Reply:
at least that could be done :)